Dockerfile.py3.aws

```diff
@@ -0,0 +1,165 @@
+# Build the docker image used to run the scripts
+# to continuously update our docker files.
+#
+# The context for this docker file should be the root of the kubeflow/testing repository.
+FROM ubuntu:18.04
+
+RUN apt-get update -y && \
+    apt-get install -y curl git python3.8 python3-pip wget && \
+    ln -sf /usr/bin/python3.8 /usr/bin/python
+
+RUN python3.8 -m pip install \
+    filelock \
+    fire \
+    google-api-python-client \
+    google-cloud \
+    google-cloud-storage \
+    junit-xml \
+    # See https://github.com/kubeflow/gcp-blueprints/issues/52#issuecomment-645446088
+    # our libs seem to break with 11.0.0
+    kubernetes==9.0.0 \
+    lint \
+    oauth2client \
+    pytest==5.4 \
+    pytest-timeout==1.4 \
+    python-dateutil \
+    retrying \
+    watchdog
+
+# Install go
+RUN cd /tmp && \
+    wget -O /tmp/go.tar.gz https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf go.tar.gz
+
+# Install gcloud
+ENV PATH=/root/go/bin:/usr/local/go/bin:/google-cloud-sdk/bin:/workspace:${PATH} \
+    CLOUDSDK_CORE_DISABLE_PROMPTS=1
+
+# Install the new version of yq which is based on go
+RUN GO111MODULE=on go get github.com/mikefarah/yq/v3
+RUN go get github.com/kelseyhightower/kube-rsa 
+
+RUN wget -q https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz && \
+    tar xzf google-cloud-sdk.tar.gz -C / && \
+    rm google-cloud-sdk.tar.gz && \
+    /google-cloud-sdk/install.sh \
+    --disable-installation-options \
+    --bash-completion=false \
+    --path-update=false \
+    --usage-reporting=false && \
+    gcloud components install alpha beta
+
+RUN gcloud components install anthoscli kpt
+
+# Install the hub CLI for git
+RUN cd /tmp && \
+    curl -LO  https://github.com/github/hub/releases/download/v2.13.0/hub-linux-amd64-2.13.0.tgz && \
+    tar -xvf hub-linux-amd64-2.13.0.tgz && \
+    mv hub-linux-amd64-2.13.0 /usr/local && \
+    ln -sf /usr/local/hub-linux-amd64-2.13.0/bin/hub /usr/local/bin/hub
+
+RUN export KUSTOMIZE_VERSION=3.2.0 && \
+    cd /tmp && \
+    curl -LO  https://github.com/kubernetes-sigs/kustomize/releases/download/v${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64 && \
+    mv kustomize_${KUSTOMIZE_VERSION}_linux_amd64 /usr/local/bin/kustomize && \
+    chmod a+x /usr/local/bin/kustomize
+
+RUN export ASM_VERSION=1.4.7-asm.0 && \
```

Comment:
istioctl is used to operator istio. Do we have any where using this?

Reply:
Removed